src/Collector.cpp

```diff
@@ -58,16 +62,17 @@ std::vector<CollectorPtr> collectorsCreate(ListOf<List> specs,
 
 // Implementations ------------------------------------------------------------
 
-void CollectorCharacter::setValue(int i, const Token& t) {
-  switch(t.type()) {
+void CollectorCharacter::setValue(int i, const Token &t) {
```

Comment:
I don't understand why it puts (what to me is) a type modifier in the object name